src/test/java/dk/kb/present/ViewTest.java

```diff
@@ -51,6 +51,17 @@ void jsonld() throws Exception {
         assertTrue(jsonld.contains("\"headline\":[{\"value\":\"Christian VIII\",\"@language\":\"da\"}]"));
     }
 
+    @Test
+    void jsonldPvica() throws Exception {
+        YAML conf = YAML.resolveLayeredConfigs("test_setup.yaml");
+        YAML radiotvConf = conf.getYAMLList(".config.collections").get(1);
+        View jsonldView = new View(radiotvConf.getSubMap("radiotv").getYAMLList("views").get(1), radiotvConf.getSubMap("radiotv").getString("origin"));
+        String pvica = Resolver.resolveUTF8String("internal_test_files/tvMetadata/df3dc9cf-43f6-4a8a-8909-de8b0fb7bd00.xml");
```

Comment:
The filenames are already defined as static fields in `XSLTPreservicaSchemaOrgTransformerTest`. If that is imported as a "static import", `RECORD_df3dc9cf` can be used instead of `internal_test_files/tvMetadata/df3dc9cf-43f6-4a8a-8909-de8b0fb7bd00.xml`. Same principle for the other tests in `ViewTest.java`.

Reply:
Good point, this could also be done on a broader scale in ds-present. I've created a jira.